file-formats/srvc/type/zif_aff_srvc_v1.intf.abap

```diff
@@ -0,0 +1,76 @@
+INTERFACE zif_aff_srvc_v1
+  PUBLIC.
+  "! <p class="shorttext">Consumer Type</p>
+  "! Consumer Type
+  "! $values {@link zif_aff_srvc_v1.data:co_cons_type}
+  TYPES ty_cons_type TYPE c LENGTH 2.
+
+  CONSTANTS:
+    BEGIN OF co_cons_type,
+      "! <p class="shorttext">OData</p>
+      "! OData
+      odata       TYPE ty_cons_type VALUE '01',
```

Comment:
Does it make sense to change the value in JSON to "OData"? Here it might make sense to use the annotation `$enumValue`.

See https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#enum-values

Reply:
Added enumValue for the constants